server/api/cart.js

```diff
@@ -0,0 +1,66 @@
+const router = require('express').Router();
+const User = require("../db/models/User");
+const Order = require("../db/models/Order");
+// const Category = require("../db/models/Category");
+// const LineItem = require("../db/models/Lineitem");
+// const Product = require("../db/models/Product");
+
+// GET /api/cart
+router.get("/", async (req, res, next) => {
+  //find user by token and get their cart
+  try {
+    const user = await User.findByToken(req.headers.authorization);
+    res.send(await user.getCart());
+  } catch (error) {
+    next(error);
+  }
+});
+
+// POST /api/addToCart
+router.post("/addToCart", async (req, res, next) => {
```

Comment:
It would probably be more RESTful to just have the route be `/` here

(so POSTing to `/api/cart` would add the req.body to the cart)